legal-api/tests/unit/services/filings/validations/test_continuation_out.py

```diff
@@ -0,0 +1,174 @@
+    )
+    filing = copy.deepcopy(FILING_HEADER)
+    filing['filing']['continuationOut'] = copy.deepcopy(CONTINUATION_OUT)
+    filing['filing']['header']['name'] = 'continuationOut'
+
+    if test_name == 'FAIL_IN_FUTURE':
+        filing['filing']['continuationOut']['continuationOutDate'] = \
+            (datetime.utcnow() + datedelta.datedelta(days=1)).strftime(date_format)
+    elif test_name == 'FAIL_EXPIRED':
+        filing['filing']['continuationOut']['continuationOutDate'] = datetime.utcnow().strftime(date_format)
+        business.cco_expiry_date = datetime.utcnow() - datedelta.datedelta(days=1)
+    elif test_name == 'FAIL_NO_CCO':
+        business.cco_expiry_date = None
+
+    err = validate(business, filing)
+
+    # validate outcomes
+    if test_name != 'SUCCESS':
+        assert expected_code == err.code
+        assert message == err.msg[0]['error']
+    else:
+        assert not err
+
+
+@pytest.mark.parametrize(
+    'test_name, expected_code, message',
+    [
+        ('FAIL_REGION_BC', HTTPStatus.BAD_REQUEST, 'Region should not be BC.'),
+        ('FAIL_INVAILD_REGION', HTTPStatus.BAD_REQUEST, 'Invalid region.'),
+        ('FAIL_INVAILD_US_REGION', HTTPStatus.BAD_REQUEST, 'Invalid region.'),
+        ('SUCCESS', None, None)
+    ]
+)
+def test_validate_foreign_jurisdiction(session, test_name, expected_code, message):
+    """Assert validate foreign jurisdiction."""
+    business = Business(
+        identifier='BC1234567',
+        legal_type='BC',
+        state=Business.State.ACTIVE,
+        founding_date=datetime.utcnow(),
+        cco_expiry_date=datetime.utcnow()
+    )
+    filing = copy.deepcopy(FILING_HEADER)
+    filing['filing']['continuationOut'] = copy.deepcopy(CONTINUATION_OUT)
+    filing['filing']['header']['name'] = 'continuationOut'
+
+    if test_name == 'FAIL_REGION_BC':
+        filing['filing']['continuationOut']['foreignJurisdiction']['region'] = 'BC'
+    elif test_name == 'FAIL_INVAILD_REGION':
+        filing['filing']['continuationOut']['foreignJurisdiction']['region'] = 'NONE'
+    elif test_name == 'FAIL_INVAILD_US_REGION':
+        filing['filing']['continuationOut']['foreignJurisdiction']['country'] = 'US'
+        filing['filing']['continuationOut']['foreignJurisdiction']['region'] = 'NONE'
+
+    err = validate(business, filing)
+
+    # validate outcomes
+    if test_name != 'SUCCESS':
+        assert expected_code == err.code
+        assert message == err.msg[0]['error']
+    else:
+        assert not err
+
+
+def test_valid_foreign_jurisdiction(session):
+    """Assert valid foreign jurisdiction."""
+    business = Business(
+        identifier='BC1234567',
+        legal_type='BC',
+        state=Business.State.ACTIVE,
+        founding_date=datetime.utcnow(),
+        cco_expiry_date=datetime.utcnow()
+    )
+    filing = copy.deepcopy(FILING_HEADER)
+    filing['filing']['header']['name'] = 'continuationOut'
+
+    for country in pycountry.countries:
```

Comment:
@severinbeauvais here is the validation for all valid jurisdiction as mentioned in schema